bot/code_review_bot/report/phabricator.py

```diff
@@ -22,7 +23,11 @@
 BUG_REPORT_URL = "https://bugzilla.mozilla.org/enter_bug.cgi?product=Developer+Infrastructure&component=Source+Code+Analysis&short_desc=[Automated+review]+THIS+IS+A+PLACEHOLDER&comment=**Phabricator+URL:**+https://phabricator.services.mozilla.com/...&format=__default__"
 
 COMMENT_FAILURE = """
-Code analysis found {defects_total}{defects_details} in the diff [{diff_id}]({phabricator_diff_url}):
+Code analysis found {defects_total} in diff [{diff_id}]({phabricator_diff_url}):
+"""
+
+COMMENT_FAILURE_OUTSIDE = """
+Code analysis found {defects_total} outside of diff [{diff_id}]({phabricator_diff_url}):
```

Comment:
Instead of splitting in two separate sentences, we could have a single one with the total number of defects (in the end they have the same level of importance, it doesn't matter if they are in the lines modified by the patch or not, it only matters if they were introduced by the patch):
```
Code analysis found {defects_total} introduced by diff [{diff_id}]({phabricator_diff_url}):
```